CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp-main.tex

```diff
@@ -1447,10 +1448,12 @@ \subsubsection*{Current Practices}
 One new model of data analysis, developed outside of HEP, maintains the
 concept of sequential reduction, but mixes interactivity with
 batch processing. These exploit new cluster management systems, most
-notably Apache Spark, which uses open-source tools contributed both by
+notably Apache Spark~\cite{Armbrust:2015:SSR:2723372.2742797}, which uses open-source tools contributed both by
 industry and the data-science community. Other products implementing the
-same analysis concepts and workflows are emerging, such as TensorFlow,
-Dask, Pachyderm, Blaze, Parsl, and Thrill. This approach can complement
+same analysis concepts and workflows are emerging, such as TensorFlow~\cite{tensorflow2015-whitepaper},
```

Comment:
Seems we've been working on the same here. You got some nice new citations. We should make sure we keep those when merging my PR with yours.